### PR TITLE
add machine config update ca local registry

### PR DESCRIPTION
**Pull request by alknopfler**
- Machine config created using templates and subenv to update the ca in master nodes
- Created bypass for hub and spokes

@jparrill needs review

### CODE REVIEW

**Comment by alknopfler** [comment]
@jparrill ready to merge after your review

File: deploy-disconnected-registry/deploy.sh [hunk 1]
```diff
@@ -59,8 +78,15 @@ function trust_internal_registry() {
 	echo ">>>> Trusting internal registry"
 	echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
```
Comment (jparrill) [inline deploy-disconnected-registry/deploy.sh:79]: Indent properly

File: deploy-disconnected-registry/deploy.sh [hunk 2]
```diff
@@ -59,8 +78,15 @@ function trust_internal_registry() {
...
 	## Update trusted CA from Helper
-	oc --kubeconfig=${TARGET_KUBECONFIG} get secret -n openshift-ingress router-certs-default -o go-template='{{index .data "tls.crt"}}' | base64 -d >/etc/pki/ca-trust/source/anchors/internal-registry-${cluster}.crt 
-    update-ca-trust extract
+    #TODO despues el sync pull secret global porque crictl no puede usar flags y usa el generico with https://access.redhat.com/solutions/4902871
+	export CA_CERT_DATA=$(oc --kubeconfig=${TARGET_KUBECONFIG} get secret -n openshift-ingress router-certs-default -o go-template='{{index .data "tls.crt"}}')
```
Comment (jparrill) [inline deploy-disconnected-registry/deploy.sh:82]: Indent properly

File: deploy-disconnected-registry/deploy.sh [hunk 3]
```diff
@@ -11,6 +11,25 @@ set -m
...
 source ./common.sh ${1}
 
+function render_file() {
+	SOURCE_FILE=${1}
+    MODE=${2}
+	if [[ $# -lt 2 ]]; then
+		echo "Usage :"
+		echo "  $0 <SOURCE FILE> <MODE> [<SPOKE_NAME>]"
+		exit 1
+	fi
```
Comment (jparrill) [inline deploy-disconnected-registry/deploy.sh]: Indent properly

**Review by jparrill — APPROVED** [review]
LGTM

**Pull request merged**